package.json

```
@@ -111,7 +111,7 @@
     "@18f/identity-stylelint-config": "1.0.0",
     "@babel/core": "7.16.7",
     "@babel/preset-env": "7.16.7",
-    "@chanzuckerberg/axe-storybook-testing": "5.0.1",
+    "@chanzuckerberg/axe-storybook-testing": "^6.3.0",
```

Should these be pinned like the others?

```suggestion
    "@chanzuckerberg/axe-storybook-testing": "6.3.0",
```